src/api/python/cvc5.pxi

```diff
@@ -2960,55 +3106,81 @@ cdef class Term:
         return elems
 
     def isUninterpretedValue(self):
+        """:return: True iff this term is a value from an uninterpreted sort."""
         return self.cterm.isUninterpretedValue()
 
     def getUninterpretedValue(self):
+        """
+	   Asserts :py:meth:`isUninterpretedValue()`.
+
+	   :return: the representation of an uninterpreted value as a pair of 
+	   its sort and its index.
+	"""
         cdef pair[c_Sort, int32_t] p = self.cterm.getUninterpretedValue()
         cdef Sort sort = Sort(self.solver)
         sort.csort = p.first
         i = p.second
         return (sort, i)
 
     def isTupleValue(self):
+        """:return: True iff this term is a tuple value."""
         return self.cterm.isTupleValue()
 
     def getTupleValue(self):
+        """
+	   Asserts :py:meth:`isTupleValue()`.
+
+	   :return: the representation of a tuple value as a vector of terms.
+	"""
         elems = []
         for e in self.cterm.getTupleValue():
             term = Term(self.solver)
             term.cterm = e
             elems.append(term)
         return elems
 
-    def getIntegerValue(self):
-        return int(self.cterm.getIntegerValue().decode())
-
     def isRealValue(self):
+        """
+	    :return: True iff this term is a rational value.
+
+	    Note that a term of kind PI is not considered to be a real value.
+	"""
         return self.cterm.isRealValue()
 
     def getRealValue(self):
-        """Returns the value of a real term as a Fraction"""
+        """
+	   Asserts :py:meth:`isRealValue()`.
+
+	   :return: the representation of a rational value as a python Fraction.
+	"""
         return Fraction(self.cterm.getRealValue().decode())
 
     def isBitVectorValue(self):
+        """:return: True iff this term is a bit-vector value."""
         return self.cterm.isBitVectorValue()
 
     def getBitVectorValue(self, base = 2):
-        """Returns the value of a bit-vector term as a 0/1 string"""
+        """
+	   Asserts :py:meth:`isBitvectorValue()`.
+
+	   :return: the representation of a bit-vector value in string representation. 
+	   Supported bases are 2 (bit string), 10 (decimal string) or 16 (hexdecimal string).
```

Comment:
Does the second line belong to the return? If not, I'd suggest to move it upwards.